src/dune_manager/dune_manager.ml

```diff
@@ -256,6 +255,26 @@ module Client = struct
       raise exn
 
   let run ?(port_f = ignore) ?(port = 0) manager =
+    let trim_thread max_size period memory =
+      let rec trim () =
+        Unix.sleep period;
+        let () =
+          match
+            let size = Dune_memory.size memory in
+            if size > max_size then (
+              Log.infof "trimming %i bytes" (size - max_size);
+              Some (Dune_memory.trim memory (size - max_size))
+            ) else
+              None
+          with
+          | Some { trimmed_files = freed, _; _ } ->
+            Log.infof "trimming freed %i bytes" freed
```

Comment:
Can we de-duplicate this log message? I.e. introduce a `log_trimmed` helper

Reply:
The messages are slightly different, it's "start trimming [at least] X bytes", and "done trimming, freed X bytes". The second value can be slightly above. Maybe the first one is superfluous, but given the process is not quite instantaneous, I'd say it's a good thing to log something before.